js/views/harvester-wizard.vue

```diff
@@ -17,7 +17,7 @@ export default {
             steps: [{
                 label: this._('Harvest as'),
                 subtitle: this._('Choose who is harvesting'),
-                component: 'publish-as',
+                component: 'publishas',
```

Comment:
I think we should, I'm tryng to gain on the produced JS size.
Each char gained in the template is welcome :)